_config/importantNews.yml

```diff
@@ -9,6 +9,12 @@
 #
 # Order news by date.
 # Don't remove past news.
+- newsId: 20210401-blog
+  newsDate: 2021-04-01
```

Comment:
maybe we should put here today not the released date?

Reply:
DONE